packages/pricing/integration-tests/__tests__/services/price-set-money-amonut-rules/index.ts

```diff
@@ -0,0 +1,300 @@
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+
+import { PriceSetMoneyAmountRulesRepository } from "@repositories"
+import { PriceSetMoneyAmountRulesService } from "@services"
+
+import { createCurrencies } from "../../../__fixtures__/currency"
+import { createMoneyAmounts } from "../../../__fixtures__/money-amount"
+import { createPriceSets } from "../../../__fixtures__/price-set"
+import { createPriceSetMoneyAmounts } from "../../../__fixtures__/price-set-money-amount"
+import { createPriceSetMoneyAmountRules } from "../../../__fixtures__/price-set-money-amount-rules"
+import { createRuleTypes } from "../../../__fixtures__/rule-type"
+import { MikroOrmWrapper } from "../../../utils"
+
+jest.setTimeout(30000)
+
+describe("PriceSetMoneyAmountRules Service", () => {
+  let service: PriceSetMoneyAmountRulesService
+  let testManager: SqlEntityManager
+  let repositoryManager: SqlEntityManager
+
+  beforeEach(async () => {
+    await MikroOrmWrapper.setupDatabase()
+    repositoryManager = await MikroOrmWrapper.forkManager()
+
+    const priceSetMoneyAmountRulesRepository =
+      new PriceSetMoneyAmountRulesRepository({
+        manager: repositoryManager,
+      })
+
+    service = new PriceSetMoneyAmountRulesService({
+      priceSetMoneyAmountRulesRepository,
+    })
+
+    testManager = await MikroOrmWrapper.forkManager()
+
+    await createCurrencies(testManager)
+    await createMoneyAmounts(testManager)
+    await createPriceSets(testManager)
+    await createRuleTypes(testManager)
+    await createPriceSetMoneyAmounts(testManager)
+    await createPriceSetMoneyAmountRules(testManager)
+  })
+
+  afterEach(async () => {
+    await MikroOrmWrapper.clearDatabase()
+  })
+
+  describe("list", () => {
+    it("should list psmar records", async () => {
```

Comment:
suggestion: We should use complete word instead of psmar, it would be more explicit for anybody looking at it. wdyt?

Reply:
if agreed, same everywhere